plutus-example/src/PlutusExample/PlutusVersion1/MintingScript.hs

```diff
@@ -18,6 +18,7 @@ import Codec.Serialise
 import Data.ByteString.Lazy qualified as LB
 import Data.ByteString.Short qualified as SBS
 
+import Plutus.Script.Utils.Typed qualified as Scripts
 import Plutus.Script.Utils.V1.Typed.Scripts qualified as Scripts
```

Comment:
In some places you have removed the V1 import and in other places not. Is there a reason for that?

Reply:
I removed it when I encounter the `Unused` warning, I'm gonna check consistency across the examples.

Reply:
Ok, it was because some import Plutus.Vx.Ledger.Api and others don't, relying on PSU.